Specs/DateTime/French/HolidayExtractor.json

```diff
@@ -1,6 +1,7 @@
 [
   {
     "Input": "Je reviendrai sur Yuandan",
+    "NotSupported":"dotnet, javascript, python, java",
```

Comment:
You should only add these to the new specs. Please don't modifiy cases that already existed previously, ok?